__tests__/app.test.js

```diff
@@ -90,115 +88,218 @@ describe("api", () => {
+              comment_count: expect.any(String),
+            });
+          });
 
-        });
+          const foundReview = reviews.find((review) => {
+            if (review.review_id === 7) {
+              return review;
+            }
+          });
 
+          expect(foundReview.comment_count).toBe("0");
         });
     });
-
-    it("400 GET - invalid review id responds with bad request msg.", () => {
+    it("200 GET - endpoint should default with DESC if there is order query.", () => {
       return request(app)
-        .get("/api/reviews/bad-request")
-        .expect(400)
+        .get("/api/reviews?category=social+deduction&sort_by=title")
+        .expect(200)
         .then(({ body }) => {
-          const serverResponseMsg = body.msg;
-          expect(serverResponseMsg).toBe("Bad request.");
-        });
-    });
+          const reviews = body.reviews;
 
-    it("404 GET - responds with msg when sent valid but non-existent path.", () => {
-      return request(app)
-        .get("/api/reviews/99999999")
-        .expect(404)
-        .then(({ body }) => {
-          const serverResponseMsg = body.msg;
-          expect(serverResponseMsg).toBe("Content not found.");
+          const reviewTitles = reviews.map((review) => {
+            return review.title;
+          });
+          expect(reviewTitles).toBeSorted({ descending: true });
         });
     });
-
-    it("201 PATCH - responds with the updated review.", () => {
+  
+    it("200 GET - returns empty array if category that exists but does not have any reviews associated with it.",()=>{
       return request(app)
-        .patch("/api/reviews/2")
-        .send({ inc_votes: 100 })
-        .expect(201)
-        .then(({ body }) => {
-          const review = body.review;
-
-          expect(review).toMatchObject({
-            review_id: 2,
-            title: 'Jenga',
-            category: 'dexterity',
-            designer: 'Leslie Scott',
-
-            owner: 'philippaclaire9',
-            review_body: 'Fiddly fun for all the family',
-            review_img_url: 'https://images.pexels.com/photos/4473494/pexels-photo-4473494.jpeg?w=700&h=700',
-            created_at: '2021-01-18T10:01:41.251Z',
-            votes: 105
-        });
-      });
-    });
-
-    it("201 PATCH - ignores other properties responds with the updated review.", () => {
+      .get("/api/reviews?category=children%27s+games")
+      .expect(200)
+      .then(({body})=>{
+        const reviews = body.reviews;
+        expect(reviews).toEqual([]);
+        
+      })
+    })
+
+    it("400 GET - returns msg if  sort by column that doesn't exist.",()=>{
       return request(app)
-        .patch("/api/reviews/2")
-        .send({ inc_votes: -100, name: "Anouk" })
-        .expect(201)
-        .then(({ body }) => {
-          const review = body.review;
-          expect(review.votes).toBe(-95);
-        });
-    });
+      .get('/api/reviews?category=does+not+exist')
```

Comment:
here we have written category instead of soryby - we need to change this path 

Status 400. invalid sort_by query, e.g. ?sort_by=bananas